libc/src/__support/FPUtil/FPBits.h

```diff
@@ -220,6 +270,9 @@ struct FPRepBase : public internal::FPLayout<fp_type> {
   }
 
   LIBC_INLINE constexpr StorageType uintval() const { return bits & FP_MASK; }
+  LIBC_INLINE constexpr StorageType set_uintval(StorageType value) {
+    return bits = (value & FP_MASK);
```

Comment:
```
bits = (value & FP_MASK);
return bits;
```

Reply:
Thx for pointing this out, I messed up. This function was not supposed to return any value.